app/guide/components/subsection.njk

```diff
@@ -34,7 +34,7 @@
       {{ button({
         "text": params.button.text,
         "classes": "nhsuk-button--secondary nhsuk-u-margin-bottom-0",
-        "href": params.button.href
+        "href": blobstoreHost + params.button.href
```

Comment:
Just need to make sure we check when we are using '/' here. The blob store host URL should not end with a '/'.